en/guide/index.md

```diff
@@ -22,57 +22,57 @@ As a framework, Nuxt.js comes with a lot of features to help you in your develop
 
 ## How it Works
 
-![Vue with Webpack and Babel](https://i.imgur.com/avEUftE.png)
+![Vue with webpack and Babel](https://i.imgur.com/avEUftE.png)
```

Comment:
As in official documentation, the product name is “webpack” without capital (even in first word)
— source : https://webpack.github.io/docs/